src/components/Certificate/certificate.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { beforeEach, describe, expect, test, vi } from "vitest";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+import Certificate from "./certificate";
+
+const username = "testUser";
+const pathname = `/members/${username}/certificate`;
+
+describe("Certificate Page", () => {
```

Comment:
```suggestion
describe("<Certificate />", () => {
```

컴포넌트에 테스트를 작성할 때는 저는 개인적으로 요렇게 표시해놓는 것을 선호하는데 어떻게 생각하는지요? 나중에 테스트가 많아지면 실패했을 때 매우 쉽게 컴포넌트에 대한 테스트가 실패했구나 알 수 있거든요.